Additions/PrivateAPI/UIDatePicker+KIFPrivateAPI.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
+
+// Private API's used for UIDatePicker.
+@interface UIDatePicker (KIFPrivateAPI)
+
+// After updating the value this is called to inform
+// the observers that the value has changed.
+- (void)_emitValueChanged;
```

Comment:
NIT: Might be better to put this under some other folder to make it clear that it is exposing/relying on underlying APIs in Apple's SDK. Maybe a new root folder like `ApplePrivateAPIs`? I think there is some stuff within `IOHIDEvent+KIF.m` that would also potentially make sense moving there as well.